### ISSUE TITLE
@font-face get parsed

### ISSUE THREAD

**Issue by laeubi** [tomyeh/ZUSS#23]
@font-face {

}

simply don't sho up in the ouput because it is parsed as a function(?) or some other ZUSS construct.

**Comment by jlangston** [comment]
One other problem is while your patch makes the @font-face not get picked up as a mixin/variable it won't parse and process mixin's within the @font-face{ } block.

**Comment by laeubi** [comment]
Yes, at the momment it is handleded as a "raw" value the is more work to do in the Parser.java, so the current support is very basic, but as a first step it prevents ZUSS from completely removing font-face definitions from the CSS...
